### PR TITLE
Adding Build instructions for Docker on Windows

### PR DESCRIPTION
**Pull request by PhRosenberger**
#### Reference to a related issue in the repository
Should solve #47 

#### Add a description
We should help building and contributing to the documentation by adding build instructions to the README.

#### Mention a member
@stefancyliax and @LukasElster what do you think on my proposed changes?

#### Check the checklist

- [x] I have performed a self-review of my own code/documentation.
- [ ] My documentation changes are related to another repository in the organization. Here is the link to the issue/repo.
- [x] My changes generate no new warnings during the documentation generation.
- [x] The existing travis ci which pushes the documentation to gh-pages passes with my changes.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,24 @@ Pipeline is broken - `OSI Validation`_
 
 |osi_sensor_model_packaging_build| - `OSI Model Packaging`_
 
-
+How To build and change Documentation locally on Windows
+--------------------------------------------------------
+- Install Docker Desktop
+- Clone this repository with submodules
+- Clone OSI and OSMP with submodules into this folder
```
Comment (stefancyliax) [inline README.rst:24]: Since this part is not intuitive, I will extent the instructions here.

**Comment by stefancyliax** [comment]
@PhRosenberger Rewrote the instructions and the whole README. Please review.

**Comment by PhRosenberger** [comment]
@stefancyliax Instructions seem fine for me.

However, I am wondering the README here shouldn't be somehow included in the Docu itself as e.g.
- **4.5 Contributing to the OSI Documentation**
  - *4.5.1 Building the Docu Locally*
  - *4.5.2 Don't hesitate, there is a review process anyways*

**Comment by stefancyliax** [comment]
> However, I am wondering the README here shouldn't be somehow included in the Docu itself as e.g.
> 
> * **4.5 Contributing to the OSI Documentation**
>   
>   * _4.5.1 Building the Docu Locally_
>   * _4.5.2 Don't hesitate, there is a review process anyways_

I'll create an issue for this. We are currently writing a central ASAM guide, that describes our working model and doc-as-code approach. I'm unsure if this should be describe there or be a part of the OSI documentation.

**Comment by PhRosenberger** [comment]
> > However, I am wondering the README here shouldn't be somehow included in the Docu itself as e.g.
> > 
> > * **4.5 Contributing to the OSI Documentation**
> >   
> >   * _4.5.1 Building the Docu Locally_
> >   * _4.5.2 Don't hesitate, there is a review process anyways_
> 
> I'll create an issue for this. We are currently writing a central ASAM guide, that describes our working model and doc-as-code approach. I'm unsure if this should be describe there or be a part of the OSI documentation.

Ok yes, good idea to take this into a new issue.

Then I would support to give this one into next CCB meeting.

**Review by stefancyliax — APPROVED** [review]

**Pull request merged**